SConstruct

```diff
@@ -502,7 +502,7 @@ libraries = {
 		"pythonEnvAppends" : {
 			"LIBS" : [ "GafferImageTest", "GafferImageBindings" ],
 		},
-		"additionalFiles" : glob.glob( "python/GafferImageTest/scripts/*" ),
+		"additionalFiles" : glob.glob( "python/GafferImageTest/scripts/*" ) + glob.glob( "python/GafferImageTest/images/*" ),
```

Comment:
All other images for test are stored in python/GafferTest/images. Should they be moved over to this new folder too?